src/steps/sse.ts

```diff
@@ -103,12 +106,35 @@ export default async function (
       }
 
       resolve(true)
+    }
+
+    const timeout = setTimeout(() => {
+      console.debug(`SSE timed out`)
```

Comment:
I left a debug log here, as I personally don't think it makes sense to mark the step as "passed" if the SSE timed out (which was the case in the current code). To make sure people notice this, I think it's acceptable to leave a warning. Maybe you'd like to use some logging utilities Step CI uses instead of `console.debug`, but I haven't found them.

Reply:
I changed the quotes for single quotes but left the log.